pages/code_schools.js

```diff
@@ -51,7 +51,7 @@ export default class CodeSchools extends React.Component {
             <div key={`${school.name}`}>
               <SchoolCard
                 hasHardwareIncluded={school.hardware_included}
-                hasHousing={school.has_housing}
+                hasHousing={school.has_housing ? school.has_housing : false}
```

Comment:
You can revert this change. We're trusting the API to give us true or false in `has_housing`

Reply:
Actually some of the responses were missing the field so I was seeing some undefined errors in console

Reply:
But let me change it back real quick

Reply:
Yes, `has_housing` is actually not in the response. SchoolCard.propTypes should list `hasHousing` as not required - no?

If it doesn't that'll be the fix.